server.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable */
+const express = require('express');
+const webpack = require('webpack');
+const webpackDevMiddleware = require('webpack-dev-middleware');
+
+const app = express();
+const config = require('./webpack.config.js');
+
+const compiler = webpack(config);
+
+app.use(
+  webpackDevMiddleware(compiler, {
+    publicPath: config.output.publicPath,
+  }),
+);
+
+app.listen(3000, () => {
+  console.log('Example app listening on port 3000!\n');
+});
```

Comment:
- [ Required ] I think it would be better if you follow the project requirements for this milestone of the project, in this milestone you are supposed to set up a new project with `webpack` that is based on the `webpack` exercise you have already completed.[Click Here for the exercise >>](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/exercises/exercise_set_up_project_with_webpack.md). I would like to suggest that you initialize `npm` in this folder and add basic commands for building `Webpack` in `development mode`, `production mode`, currently, your project does not have a `package.json` file that lists dependencies of your application. Please make sure you follow all the steps in the previous exercise while setting up the Webpack Boilerplate.

## Project requirements
![Screenshot from 2023-03-30 02-19-46](https://user-images.githubusercontent.com/16392046/228689292-735cf9d6-7a03-45a4-89ff-093fbef53c66.png)

Reply:
I do have package.json locally. I did not know if it was required to upload it too @Felix45